### PR TITLE
RespaAdmin: Add debug-toolbar

### PR DESCRIPTION
**Pull request by JsseL**
Makes development a bit nicer.

### CODE REVIEW

File: respa/settings.py [hunk 1]
```diff
@@ -108,6 +108,13 @@
     'django.middleware.security.SecurityMiddleware',
 ]
 
+if DEBUG:
+    INSTALLED_APPS.append('debug_toolbar')
+    INTERNAL_IPS = ['127.0.0.1', '172.20.0.1',]
```
Comment (suutari-ai) [inline respa/settings.py]: The 172.20.0.1 won't be the same on different machines.

Comment (JsseL) [reply respa/settings.py]: Ah.. sorry, forgot by accident. I guess we'll just have to leave it out and expect that people know to add their own IPs there.

**Comment by suutari-ai** [comment]
Actually. This is still missing the required python package. Though that should not probably be installed when running the tests. Maybe change the tests to run with DEBUG=0?

**Comment by JsseL** [comment]
@vikoivun Shouldn't tests be ran without DEBUG?

**Comment by JsseL** [comment]
@vikoivun Found the problem now. We defaulted to `DEBUG=True` in settings. Changed that since it ain't good practice. Let's see if the tests pass now.

**Comment by JsseL** [comment]
@suutari-ai @dvainio merge?

**Comment by JsseL** [comment]
Scheisse.. accidentally closed. Well no problem there..

**Pull request merged**